CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 <!-- ## Unreleased -->
 
+- Add `collate` option for collating results tables by measurement (when
```

Comment:
How about `partition: "measurement"`?

I also thought about names `compare` and `group-by`, but I'm leaning towards `partition` because it seems to best describe "the thing you want to do to the table after you see that it's too big".

Also I'm thinking the value should be `measurement` instead of `true` so that we have more room to expand with more partitioning options in the future.

WDYT?

Reply:
Oh, that's nice. So conceptually we could add something like `partition: "browser"` to partition the results table based on browser? And next level, make it an array to partition on multiple dimensions, i.e. browser and measurement?

Reply:
Sounds good. Was out yesterday but will try and get an update up soon.

Reply:
I've addressed the feedback. All instances of `collate: true` have been replaced by `partition: "measurement"`.
I also added documentation to the README for the new option.